src/timings/Timings.php

```diff
@@ -299,4 +306,43 @@ public static function getEventHandlerTimings(string $event, string $handlerName
 
 		return self::$eventHandlers[$event][$handlerName];
 	}
+
+	public static function getAsyncTasksProgressUpdateTimings(AsyncTask $task) : TimingsHandler{
+		$taskClass = get_class($task);
+		if(!isset(self::$asyncTasksProgressUpdate[$taskClass])){
+			self::$asyncTasksProgressUpdate[$taskClass] = new TimingsHandler(
+				"AsyncTask Progress Update - " . $taskClass,
```

Comment:
```suggestion
				"AsyncTask - Progress Updates - " . self::shortenCoreClassName($taskClass, "pocketmine\\"),
```

Reply:
I wonder if we might be able to get away with removing the `AsyncTask` prefix here. It makes the timings output unnecessarily noisy in tree mode.

Reply:
Deleting it would certainly reduce the number of entries in Timings, but it could lead to confusion if the class name isn't so explicit and the fact that it's an AsyncTask behind it isn't self-evident.

Reply:
Or we can start with a naming scheme like this: ``SomeShortenClass::onCompletion()`` / ``SomeShortenClass::onError()`` / ``SomeShortenClass::onProgressUpdates()``

Reply:
Fair enough, probably better to let the prefix stay then.